### DISCUSSION TITLE
SIZE OF IMAGES

### DISCUSSION THREAD

**Discussion by marcootaviomello** [arkypita/LaserGRBL#1056] [Q&A]
guys, good night.
I am a beginner, I started testing with the laserGRBL today, my machine is working, recording and cutting, but when I put measurements to generate the image to be recorded, these measurements are not preserved.

**Comment by arkypita** [comment]
If you mean that the engraving comes out in a different size you should configure $100 and $101 in grbl configuration
http://lasergrbl.com/wp-content/uploads/2020/05/Grbl-Configuration-ENG.pdf
